lib/mbta.js

```diff
@@ -1,11 +1,24 @@
 'use strict'
 
 // Code here.
+const subLines = {
+  'Red': ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard',
+    'Porter', 'Davis', 'Alewife'],
+  'Green': ['Government Center', 'Park Street', 'Boylston', 'Arlington',
+    'Copley', 'Hynes', 'Kenmore'],
+  'Orange': ['North Station', 'Haymarket', 'Park Street', 'State',
+    'Downtown Crossing', 'Chinatown', 'Back Bay', 'Forest Hills']
+
+}
 
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
```

Comment:
I wasn't sure if I missed this in class, but just kind of assumed this was shorthand for a function. Javascript loves shorthand I see.